mapsforge-map/src/main/java/org/mapsforge/map/rendertheme/RenderContext.java

```diff
@@ -43,15 +46,15 @@ public class RenderContext {
     public final CanvasRasterer canvasRasterer;
 
     // Data generated for the rendering process
-    private List<List<ShapePaintContainer>> drawingLayers;
-    public final List<MapElementContainer> labels;
-    public final List<List<List<ShapePaintContainer>>> ways;
+    private List<List<ShapePaintContainer>> drawingLayer;
```

Comment:
@Sublimis Why in the singular?

Reply:
https://github.com/mapsforge/mapsforge/blob/dc4707532c3ee0cd9af8862ddca74ac9f32c8033/mapsforge-map/src/main/java/org/mapsforge/map/rendertheme/RenderContext.java#L68

In fact this is also a typo, it should also be singular

Reply:
> In fact this is also a typo, it should also be singular

@Sublimis Then we should also change this.